src/common/header.css

```diff
@@ -1,12 +1,16 @@
 .header-container {
-    position: fixed;
+    position: sticky;
```

Comment:
ah interesting, so the intended behavior here is for the squarespace header to scroll off the page, and then this header sticks to top? (just verifying, if correct, this looks good)